src/Console/SessionsCommand.php

```diff
@@ -17,13 +17,20 @@
 
 namespace Colopl\Spanner\Console;
 
+use Colopl\Spanner\Connection;
 use Colopl\Spanner\Connection as SpannerConnection;
+use Colopl\Spanner\Session;
 use Illuminate\Console\Command;
 use Illuminate\Database\DatabaseManager;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Str;
+use RuntimeException;
 
 class SessionsCommand extends Command
 {
-    protected $signature = 'spanner:sessions {connections?* : The database connections to query}';
+    protected $signature = 'spanner:sessions {connections?* : The database connections to query}
+               {--sort= : Name of column to be sorted (default: LastUsedAt)}
+               {--order= : sort order as "ASC" or "DESC" (default: DESC)}';
```

Comment:
>You may assign default values to options by specifying the default value after the option name.
https://laravel.com/docs/9.x/artisan#options-with-values

```suggestion
               {--sort=LastUsedAt : Name of column to be sorted}
               {--order=DESC : sort order as "ASC" or "DESC"}';
```

Reply:
fixed.